src/controller/petitions.py

```diff
@@ -1,65 +1,105 @@
 from fastapi_sqlalchemy import db
+from datetime import datetime
 from sqlalchemy.sql import func
 
 from model.Schema import Petition
-from model.Database.petitions import Petitions
-from model.Database.agreements import Agreements
+from model.Database import Petitions, Agreements
 
 
-def new_petition(data: Petition.Create):
-    db_petition = Petitions(
-        title=data.title,
-        contents=data.contents,
-        proposal=data.proposal,
-        petitioner=str(data.petitioner),
-    )
-    con = db.session
-    con.add(db_petition)
-    con.commit()
-    con.refresh(db_petition)
-    return {"id": db_petition.petition_id}
+class PetitionController:
+    def __init__(
+        self,
+        id: int = None,
+        status: int = None,
+        petitioner: int = None,
+        title: str = None,
+        contents: str = None,
+        proposal: str = None,
+        agreed: int = None,
+        created_at: datetime = None,
+        end_at: datetime = None,
+        agreeable: bool = None,
+        answer: str = None,
+        answered_at: datetime = None,
+        answered_by: str = None,
+    ):
+        self.id = id
+        self.status = status
+        self.petitioner = petitioner
+        self.title = title
+        self.contents = contents
+        self.proposal = proposal
+        self.agreed = agreed
+        self.created_at = created_at
+        self.end_at = end_at
+        self.agreeable = agreeable
+        self.answer = answer
+        self.answered_at = answered_at
+        self.answered_by = answered_by
 
+    def create(self):
+        db_petition = Petitions(
```

Comment:
여기에 `self.title`이나 `self.contents`처럼 필수인자값인경우에도, 값을 None을 가지고 있을 수 있으니,
```suggestion
       try:
            db_petition = Petitions(
```
try except문을 넣어주는것이 낫다고 여겨집니당

Reply:
이거 공백갯수 이상하니까 suggetion은 누르지말아주세용

Reply:
요청에서 값을 제대로 넣지 않을 때, 라우터에서 타입이 맞지 않아 422 Unprocessable Entity 에러를 뱉기 때문에 그대로 두었는데, 여기서도 schema 모델 이용해서 검사 한번 해주어도 될거 같네요 :D